calculator.rb

```diff
@@ -0,0 +1,63 @@
+#combine all if blocks into one method
+def process_operator(operation, numbers)
+  if operation == "add" || operation == "+"
+    return do_math("+", numbers[0], numbers[1])
+  elsif operation == "subtract" || operation == "-"
+    return do_math("-", numbers[0], numbers[1])
+  elsif operation == "multiply" || operation == "*"
+    return do_math("*", numbers[0], numbers[1])
+  elsif operation == "divide" || operation == "/"
+    return do_math("/", numbers[0], numbers[1])
+  elsif operation == "modulo" || operation == "%"
+    return do_math("%", numbers[0], numbers[1])
+  end
+end
+
+
+def get_number(operation)
+  begin
+    puts "What's your first number?"
```

Comment:
I like making getting a number from the user its own method - good organization.